Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.11.1-alpine
+
+RUN mkdir app
+
+WORKDIR /app
+
+COPY . /app
+
+RUN npm install --production
```

Comment:
Great practice of using the production flag to ignore install of dev dependencies!

Reply:
Thank you! 😁